content/docs/command-reference/metrics/index.md

```diff
@@ -47,8 +47,10 @@ Path          Metric    HEAD      workspace  Change
 metrics.json  AUC       0.763981  0.801807   0.037826
 ```
 
-`dvc metrics` subcommands by default use the metrics files specified in
-`dvc.yaml` (if any), for example `summary.json` below:
+`dvc metrics` subcommands can be used on any
+[valid metrics files](#supported-file-formats). By default they use the metrics
+files specified in `dvc.yaml` (if there are any), for example `summary.json`
```

Comment:
I changed `if any` tp `if there are any` - sounded better to me, but it might be just me, I can roll this back :)

Reply:
Nice, thanks. This works perfectly in terms of meaning. I'll rephrase the wording a little for readability.